dcrwallet.go

```diff
@@ -181,8 +176,9 @@ func walletMain() error {
 	// before exiting.  Interrupt handlers run in LIFO order, so the wallet
 	// (which should be closed last) is added first.
 	addInterruptHandler(func() {
+		loader.StopTicketPurchase()
```

Comment:
please assign the error to _ so it's explicit that it is being ignored (comment would be nice too)